docs/contracts/v4/guides/15-single-sided-liquidity.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: Single-sided Liquidity
+---
+
+In this example we will create a single-side liquidity position a Buy Limit style range order that  swaps token1 for token0 as the market price falls into your defined tick range.
+
+Fetch the current pool price (sqrtPriceX96)
+
+Compute a -5% target price and align it with a usable tick
+
+Calculate the required liquidity using only token1
+
+Approve the correct token and use mint() to create a position
+
+Monitor and fine-tune the amount1Max buffer for slippage tolerance
+
+## Introduction
+
+This guide will cover how single-side liquidity provisioning can be used to execute **Limit Orders** on Uniswap V4 Pools.
+
+
+This guide will **cover**:
+
+1. Understanding Range Orders
+2. Calculating our Tick Range
+3. Creating a single-side liquidity position
+5. Closing the Limit Order
+
+Before working through this guide, consider checking out the Range Orders [concept page](https://docs.uniswap.org/concepts/protocol/range-orders) to understand how Limit orders can be executed with Uniswap v4.
+
+:::info
+This guide is built using the [v4-template repository](https://github.com/uniswapfoundation/v4-template) and utilises the Config and Constants files as imports for contract addresses.
+:::
+
+## Understanding Range Orders
+
+If you have read the [Range Order Concept page](../../../../concepts/protocol/range-orders.md), you can skip this section.
+
+Positions on a v4 Pool are always created with a Tick range in which their liquidity is accessible to swaps on the Pool.
+
+If we consider the structure of mint function in the Position Manager contract, that helps mint a new position.
+
+```solidity
+function _mint(
+    PoolKey calldata poolKey,
+    int24 tickLower,
+    int24 tickUpper,
+    uint256 liquidity,
+    uint128 amount0Max,
+    uint128 amount1Max,
+    address owner,
+    bytes calldata hookData
+)
+```
+
+We see that a position only stores a single `liquidity` value, and a `tickLower` and `tickUpper` value that define the range in which the liquidity of the Position can be utilised for Swaps.
+The actual **amount** of `token0` and `token1` that a Pool owes the Position owner is calculated from the parts of the liquidity position that are to the left and right of the current Tick.
+Liquidity left of the current Tick is denominated in `token0` and liquidity right of the current Tick is denominated in `token1`.
+
+If a new Position is created and the Tick Range of the position does not include the current Tick of the Pool, only one of the two Tokens in the Pool can be provided.
+
+We will call this a **Single Side Liquidity Position**.
```

Comment:
Im thinking we should define single-sided positions much earlier in the document